css/style.css

```diff
@@ -0,0 +1,229 @@
+* {
+  box-sizing: border-box;
+}
+
+.row {
+  background: url(https://images.unsplash.com/photo-1508848484850-f696c839aabd?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=e4a425b35d743e14ed00824b39731186&auto=format&fit=crop&w=1600&q=80);
+  background-size: cover;
+  background-position: right bottom;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-origin: content-box;
+  width: 100%;
+}
+
+.row::after {
+  content: '';
+  clear: both;
+  display: block;
+}
+
+[class*='col-'] {
+  float: left;
+  padding: 15px;
+}
+
+html {
+  font-family: 'Kalam', sans-serif;
+}
+
+html,
+body {
+  margin: 0;
+  height: 100%;
+}
+
+.header {
+  background-color: #64827a;
+  padding: 15px;
+}
+
+main .row {
+  padding: 15px;
+  text-align: center;
+}
+
+.pizza-item section {
+  padding: 15px;
+  border: 2px solid #fff;
+}
+
+.pizza-info section {
+  display: flex;
+  justify-content: space-between;
+  color: #fff;
+  background-color: #64827a;
+  opacity: 0.7;
+  filter: alpha(opacity=60); /* For IE8 and earlier*/
+}
+
+.pizza-img {
+  padding: 25px;
+  background-color: #64827a;
+  opacity: 0.7;
+  filter: alpha(opacity=60); /* For IE8 and earlier*/
+}
+
+.header ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+  font-size: 18px;
+  /*   opacity: 0.6;
+filter: alpha(opacity=60); /* For IE8 and earlier */
```

Comment:
Avoid support of such __old__ browsers. I suggest you to develop fast for first point, and very flexible for second. But it all is up to you.

Please, avoid of commented code in the repo.